samples/snippets/csharp/VS_Snippets_CLR_System/system.globalization.charunicodeinfo.getnumericvalue/cs/getnumericvalue1.cs

```diff
@@ -15,8 +15,7 @@ private static void Overload1()
       int utf32 = 0x10107;       // AEGEAN NUMBER ONE
       string surrogate = Char.ConvertFromUtf32(utf32);
       foreach (var ch in surrogate)
-         Console.WriteLine("U+{0:X4}: {1}    ", Convert.ToUInt16(ch),
-                           System.Globalization.CharUnicodeInfo.GetNumericValue(ch));
+         Console.WriteLine($"U+{(ushort)ch:X4}: {System.Globalization.CharUnicodeInfo.GetNumericValue(ch)}    ");

```

Comment:
We can do `using` here, but I'm not sure what's preferred for docs

Reply:
This one should stay with the fully qualified namespace.

In docs, we generally prefer the `using`, but will use the fully qualified namespace to support runnable snippets in the browser. `System.Globalization` isn't added by default when the snippet is plugged into the (not visible) `Main` method.